EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -10609,30 +10609,30 @@ QUEST_LV_0100_20151022_010583	Romuva of Suppression is complete but the Guide Ow
 QUEST_LV_0100_20151022_010584	Speak to Mardas
 QUEST_LV_0100_20151022_010585	The Guide Owl Sculpture asks you to speak with Mardas.
 QUEST_LV_0100_20151022_010586	Mardas came up with a solution after a long discussion with the Owl Chief Sculpture. Listen to what Mardas has to say.
-QUEST_LV_0100_20151022_010587	Eliminate the evil energy that is corrupting the forest
-QUEST_LV_0100_20151022_010588	Mardas claims that the evil energy corrupting the forest needs to be taken out first. Install Romuva ofd Suppression around it to stop the evil energy.
+QUEST_LV_0100_20151022_010587	Eliminate the evil energy that is corrupting the forest
+QUEST_LV_0100_20151022_010588	Mardas claims that the evil energy corrupting the forest needs to be taken out first. Place the Romuva of Suppression around it to stop the evil energy.
 QUEST_LV_0100_20151022_010589	Report to Mardas
 QUEST_LV_0100_20151022_010590	You've eliminated all the evil energy. Report this back to Mardas.
 QUEST_LV_0100_20151022_010591	All the evil energy is gone. Speak with Mardas on what to do next.
 QUEST_LV_0100_20151022_010592	Avoid the Surveillance and Infiltrate
 QUEST_LV_0100_20151022_010593	Mardas explains that in order to approach Black Smoke that is draining the forest energy. You must avoid getting detected by Surveillance Eyes. Once you infiltrate the place, disable the magic circle that controls the eyes.
 QUEST_LV_0100_20151022_010594	Eliminate Black Smoke
-QUEST_LV_0100_20151022_010595	Eliminate Black Smoke that is draining the energy of the forest.
-QUEST_LV_0100_20151022_010596	You've eliminated Black Smoke. Report to Mardas.
+QUEST_LV_0100_20151022_010595	Eliminate Black Smoke that is draining the energy of the forest.
```

Comment:
No "the" for black smoke here? Also i now realize it would be redundant if the mission name was eliminate the black smoke and then it starts with eliminate the black smoke.

Could we...change the first to Mysterious black smoke or something?